iidm-xml-converter/src/main/java/eu/itesla_project/iidm/xml/XMLImporter.java

```diff
@@ -79,6 +79,10 @@ private String findExtension(ReadOnlyDataSource dataSource) throws IOException {
     @Override
     public boolean exists(ReadOnlyDataSource dataSource) {
         try {
+            //skip exploded CIM1 files (assumption: CIM1 files' extensions are _EQ, _TP. _SV, _ME)
+            if (dataSource.getBaseName().endsWith("_EQ") || dataSource.getBaseName().endsWith("_TP") || dataSource.getBaseName().endsWith("_SV") || dataSource.getBaseName().endsWith("_ME") ) {
+                return false;
```

Comment:
Same kind of remark. It doesn't make nay sense to refer to CIM data in XML importer